pkg/base1/Makefile.am

```diff
@@ -16,6 +16,8 @@ nodist_base_DATA = \
 	pkg/base1/jquery.min.js.gz \
 	pkg/base1/cockpit.min.js.gz \
 	pkg/base1/bundle.min.js.gz \
+	pkg/base1/react.min.js.gz \
+	pkg/base1/redux.min.js.gz \
```

Comment:
Lets put this redux.js file the vms package for now. We're actively moving stuff out of the base1 package until we can reliably commit to it being stable long term.